### DISCUSSION TITLE
Util:CloseApplication not working

### DISCUSSION THREAD

**Discussion by Steve0212a** [wixtoolset/issues#7371] [Questions]
I am using the newly release v4 WIX - Congrats on getting that released!

I am trying to close two applications before upgrade or uninstall.

Inside my Package node, I added this:
```
    <util:CloseApplication Id="CloseUI" Target="prog1.exe" CloseMessage="no" RebootPrompt="no" />
    <util:CloseApplication Id="CloseUtility" Target="prog2.exe" CloseMessage="no" RebootPrompt="no" />
```

However, when I do an upgrade, nothing is closed.  I do see this in my log file:
```
MSI (s) (48:C0) [13:05:33:950]: Doing action: Wix4CloseApplications_X64
Action ended 13:05:33: CreateFolders. Return value 0.
MSI (s) (48:50) [13:05:33:952]: Invoking remote custom action. DLL: C:\Windows\Installer\MSI92DC.tmp, Entrypoint: WixCloseApplications
Action start 13:05:33: Wix4CloseApplications_X64.
MSI (s) (48:C0) [13:05:33:959]: Doing action: InstallFiles
Action ended 13:05:33: Wix4CloseApplications_X64. Return value 1.
```

Am I missing something?  I confirmed that my exe names are correct by running taskkill.exe with the exe names to verify they are killed.

thx

**Comment by barnson** [comment]
You turned off the default behavior, so there's nothing for it to do.

**Comment by Steve0212a** [comment]
LOL - I misunderstood the parameter CloseMessage.  thx
